packages/lesswrong/server/dialogues/cron.ts

```diff
@@ -21,3 +26,47 @@ addCronJob({
     })
   }
 });
+
+async function checkActiveUserSession(userId:string, documentId:string) {
+  const ckEditorDocumentId = getCKEditorDocumentId(documentId, userId, "edit");
+
+  const response = await ckEditorApi.getAllConnectedUserIds(ckEditorDocumentId)
+  const connectedUsers = JSON.parse(response);
```

Comment:
Double-checking that you've checked that the `/collaborations/${documentId}/users` endpoint returns an array (rather than, say, an object with a field in it, which is itself an array)?

Reply:
This feature has been tested with the live endpoint.